js/contextMenu.js

```diff
@@ -0,0 +1,37 @@
+document.getElementById('tabs').addEventListener('contextmenu', (e) => {
+  e.preventDefault();
+  e.stopPropagation();
+
+  if (e.target.classList.contains('tab-input')){
+
+    remote = electron.remote, Menu = remote.Menu;
```

Comment:
`remote` is already defined in default.js, so you don't need to re-define it here.